docs/pages/guides/upgrading/1.0.mdx

```diff
@@ -0,0 +1,196 @@
+---
+meta:
+  title: "Upgrading to 1.0"
+  parentTitle: "Guides"
+  description: "Guide to upgrade to Liveblocks version 1.0"
+---
+
+```bash
+npm update @liveblocks/client@1.0 @liveblocks/node@1.0
+```
+
+The major release of 1.0 marks the maturity of Liveblocks. The primary change
+you should note is the transition to making the `userId` mandatory in the
+`authorize` option. This change is in line with our
+[new pricing model](https://liveblocks.io/pricing).
+
+Let’s take a look!
+
+## Upgrading to Liveblocks 1.0
```

Comment:
Do we really need this title here? Isn't this the title of the page already?

Reply:
I thought it would be nice to have in the side nav to separate improvements/ changes from upgrade steps.  I can remove, or rename to `Upgrade Steps` or `Package Specific Upgrade Steps`